### DISCUSSION TITLE
1000 page limit inquiry

### DISCUSSION THREAD

**Discussion by j1mc** [retypeapp/retype#666] [General]
I'm curious about the 1000 page limit for the 'pro' and 'enterprise' licensed versions of retype. Can there be any exceptions, particularly for the 'enterprise' version of the software? We're looking to break-out some multi-step tasks into multiple pages, and fear hitting our 1000 page limit rather quickly.

**Comment by geoffreymcgill** [comment]
Hello @j1mc,

We can "stack" Pro and/or Enterprise licenses to add more pages. For example, if you require a 2000 page limit, then two Pro licenses can be purchased and "stacked" together. 

If you have a Pro or Enterprise license and are banging up against the 1000 page limit, just send an email to hello@retype.com and we can send back a new license with a little more room.

For Retype Enterprise, the 1000 page limit is per project. If you have multiple projects within your organization, each can have up to 1000 pages.

As well, we very rarely see projects that exceed 1000 pages. The vast majority of projects fall into the 20-100 page range. 

Hope this helps.

**Comment by j1mc** [comment]
Thank you for the reply, and happy holidays to you and the rest of the retype team. This answers my question, and I will close this discussion.
